src/services/legal.services.ts

```diff
@@ -186,4 +186,28 @@ export default class LegalServices {
     }
     return toReturn
   }
+
+  /**
+   * Fetches parties list.
+   * @param businessId the business identifier (aka entity inc no)
+   * @returns a promise to return the parties from the response
+   */
+  static fetchParties (businessId: string): Promise<any> {
```

Comment:
Please move this up in this class (with the other fetchers). Just to keep things together :)